vcd/resource_vcd_nsxt_network_dhcp_binding.go

```diff
@@ -108,15 +108,15 @@ func resourceVcdNsxtDhcpBinding() *schema.Resource {
 						"sntp_servers": {
 							Optional:    true,
 							Type:        schema.TypeSet,
-							Description: "",
+							Description: "Set of SNTP servers",
 							Elem: &schema.Schema{
 								Type: schema.TypeString,
 							},
 						},
-						"dns_servers": {
+						"domain_names": {
```

Comment:
Note to reviewers - this was undocumented field that we couldn't test at the time of implementation due to not having IPv6 support throughout Terraform